gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -93,9 +93,7 @@ fluid.defaults("gpii.flowManager.cloudBased", {
 // Add the liveness (/ready) and the readiness (/health) endpoints only when the cloud based flow manager is running as a
 // standalone WITHOUT having the preferences server running on the same server.
 gpii.flowManager.cloudBased.cloudStatusGradeFinder = function (preferencesServer) {
-    if (!preferencesServer) {
-        return "gpii.flowManager.cloudBased.cloudStatus";
-    }
+    return !preferencesServer ? "gpii.flowManager.cloudBased.cloudStatus" : undefined;
```

Comment:
While we are at it, express this using contextAwareness as well. 
When we do write logic, I think there is a mild stylistic preference for always choosing unnegated conditions - e.g. if (condition) rather than if (!condition).